docs/for_developers.md

```diff
@@ -113,6 +113,8 @@ flowchart LR
 
     To include debugging messages from flask, add `--debug` option to the command.
 
+    While running backend and frontend, developer may encounter [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) errors in web browser's console that prevent communication to the server. To get rid of CORS add ```FLASK_USE_CORS=True``` in .env file. Also pay attention if your frontend runs on ```http://localhost:3000```, because right now cors_config in application.py specifies this url .
```

Comment:
```suggestion
    While running backend and frontend, developer may encounter [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) errors in web browser's console that prevent communication to the server. To resolve these CORS issues, one should set FLASK_USE_CORS=True in the `.env` file.  Also pay attention if your frontend runs on ```http://localhost:3000```, because right now cors_config in application.py specifies this url .
```